sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/SclService.java

```diff
@@ -53,17 +58,17 @@ public static SclRootAdapter updateHeader(@NonNull SCL scd, @NonNull HeaderDTO h
         boolean hUpdated = false;
         String hVersion = headerDTO.getVersion();
         String hRevision = headerDTO.getRevision();
-        if(hVersion != null && !hVersion.equals(headerAdapter.getHeaderVersion())){
+        if (hVersion != null && !hVersion.equals(headerAdapter.getHeaderVersion())) {
             headerAdapter.updateVersion(hVersion);
             hUpdated = true;
         }
 
-        if(hRevision != null && !hRevision.equals(headerAdapter.getHeaderRevision())){
+        if (hRevision != null && !hRevision.equals(headerAdapter.getHeaderRevision())) {
             headerAdapter.updateRevision(hRevision);
             hUpdated = true;
         }
 
-        if(hUpdated && !headerDTO.getHistoryItems().isEmpty()){
+        if (hUpdated && !headerDTO.getHistoryItems().isEmpty()) {
             headerAdapter.addHistoryItem(
                     headerDTO.getHistoryItems().get(0).getWho(),
```

Comment:
This is related to this issue #6. The user add manually the attributes `who`, `what` and `why`. Only first because those attributes are optional.
Here, there are no change of functionality it is just about a formatting code and i think that is not necessary.

Reply:
For issue #6 i agree but the other reasons should be checked because here we don't talk about user actions, updates could come from file or other element. The optional nature don't avoid having many optional hitems.
To be discussed ...

Reply:
This part has not been modified in this US (only rearranged code), but you're wright it could be interesting to have more than one history item

Reply:
Ok, it will be implemented for an update not for an creation. Implementation is good ==> resolved